run-tests-with-build-optawebs-parent/optawebs-project-sources-test/optaweb-vehicle-routing-test-script-openshift.sh

```diff
@@ -32,7 +32,7 @@ readonly project_basedir=$1
 }
 
 #open street map git url
-readonly test_osm_data_url="https://github.com/kiegroup/optaweb-vehicle-routing/raw/master/optaweb-vehicle-routing-standalone/data/openstreetmap/planet_12.032%2C53.0171_12.1024%2C53.0491.osm.pbf"
+readonly test_osm_data_url="https://github.com/kiegroup/optaweb-vehicle-routing/raw/main/optaweb-vehicle-routing-standalone/data/openstreetmap/planet_12.032%2C53.0171_12.1024%2C53.0491.osm.pbf"
```

Comment:
Should it be main branch here also? Wouldn't it make sense to use 8.11.x or whatever branch matches?
This being a script, we could potentially use a system variable holding the branch information. If this is premature optimization (dealing with improbable situation) then please dismiss this comment.